.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,16 +30,10 @@
 
 
 
-## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
```

Comment:
change it to:
## Mobile & Desktop Screenshots or GIFs(If Template have animations)

Reply:
This is removed one